resources/application-connector/charts/connection-token-handler/templates/tests/role-binding.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Release.Name }}-connection-token-handler-tests
```

Comment:
The same thing I mentioned in `test-acceptance.yaml` file. For consistency I suggest to use Chart values everywhere it is possible